### ISSUE TITLE
TM1py REST API

### ISSUE THREAD

**Issue by VadymS77** [cubewise-code/tm1py#582]
Hi,
have you example python scripts with using TM1py.Services.RestService  with GET POST PUT methods 
I will be very grateful

**Comment by rkvinoth** [comment]
Could you refine your question a little bit?
What is your usecase?

**Comment by VadymS77** [comment]
Hi rkvinoth, thanks for response
I wrote here as there is a problem
I need to create circle between TM1 and git using python rest api scripts
I have four rest scripts (...,Pull.Execute and Post.Execute are doesn't work in python, other scripts are working GitInit/tm1project/Metadata/...etc )
first of all, scripts on Postman/VSCode (on server and local machine) are working fine! Objects are going to a git branch and came back to another tm1 models.

problem on python script api(/v1/GitPlans('plan}}')/tm1.Execute)
plan is empty, realy enpty, result of execution sript  "/api/v1/GitPlans":
List of plans:  {'@odata.context': '$metadata#GitPlans', 'value': []}

I'm trying another way to use class of TM1Py. I found example for it use 
https://tm1py.readthedocs.io/en/stable/TM1py.Services.html#module-TM1py.Services.RESTService
class TM1py.Services.RESTService.RESTService(**kwargs)¶
this question is higher priority for me.
